airbyte-integrations/connectors/source-oracle/src/test/java/io/airbyte/integrations/source/oracle/OracleJdbcSourceAcceptanceTest.java

```diff
@@ -111,6 +116,35 @@ protected void incrementalDateCheck() throws Exception {
         Lists.newArrayList(getTestMessages().get(1), getTestMessages().get(2)));
   }
 
+  protected AirbyteCatalog getCatalog(final String defaultNamespace) {
+    return new AirbyteCatalog().withStreams(List.of(
+        CatalogHelpers.createAirbyteStream(
+            TABLE_NAME,
+            defaultNamespace,
+            Field.of(COL_ID, JsonSchemaType.NUMBER),
```

Comment:
this should be an integer :/ can you make an issue to figure out why this is still being discovered as number?

Reply:
sure, will do :)

Reply:
@mkhokh-33 do you have a link to this issue? I'm writing up a doc for the types project and want to add a link

Reply:
> @mkhokh-33 do you have a link to this issue? I'm writing up a doc for the types project and want to add a link

hi, @edgao 
here it is - https://github.com/airbytehq/airbyte/issues/15328